src/Evolution/Systems/Cce/System.hpp

```diff
@@ -0,0 +1,116 @@
+ * and one evolution equation that governs the evolution of one hypersurface to
+ * the next.
+ *
+ * The mathematics of CCE are intricate, and SpECTRE's version implements a
+ * number of tricks and improvements that are not yet present in other contexts.
+ * For introductions to CCE generally, see papers \cite Bishop1997ik,
+ * \cite Bishop1998uk, and \cite Barkett2019uae. Here we do not present a full
+ * description of all of the mathematics, but instead just provide a high-level
+ * roadmap of the SpECTRE utilities and how they come together in the CCE
+ * system. This is intended as a map for maintainers of the codebase. If you are
+ * interested primarily in running SpECTRE CCE to obtain waveform data, please
+ * see the executable documentation.
+ *
+ * First, worldtube data from a completed or running GH evolution must be
+ * translated to Bondi spin-weighted scalars at the extraction sphere. Relevant
+ * utilities for this conversion are `Cce::WorldtubeDataManager`,
+ * `Cce::ReducedWorldtubeDataManager`, `Cce::create_bondi_boundary_data`.
+ * Relevant parts of the parallel infrastructure are `Cce::H5WorldtubeBoundary`,
+ * `Cce::Actions::BoundaryComputeAndSendToEvolution`,
+ * `Cce::Actions::RequestBoundaryData`, and
+ * `Cce::Actions::ReceiveWorldtubeData`.
+ *
+ * The first hypersurface must be initialized with some reasonable starting
+ * value for the evolved Bondi quantity \f$J\f$. There isn't a universal perfect
+ * prescription for this, as a complete description would require, like the
+ * Cauchy initial data problem, knowledge of the system arbitrarily far in the
+ * past. A utility for assigning the initial data is `Cce::InitializeJ`.
+ *
+ * SpECTRE CCE is currently unique in implementing an additional gauge transform
+ * after the worldtube boundary data is derived. This is performed to obtain an
+ * asymptotically well-behaved gauge that is guaranteed to avoid logarithmic
+ * behavior that has plagued other CCE implementations, and so that the
+ * asymptotic computations can be as simple, fast, and reliable as possible.
+ * Relevant utilities for the gauge transformation are
+ * `Cce::GaugeAdjustedBoundaryValue` (see template specializations),
+ * `Cce::GaugeUpdateTimeDerivatives`, `Cce::GaugeUpdateAngularFromCartesian`,
+ * `Cce::GaugeUpdateJacobianFromCoordinates`, `Cce::GaugeUpdateInterpolator`,
+ * `Cce::GaugeUpdateOmega`, and `Cce::InitializeGauge`.
+ *
+ * Next, the CCE system must evaluate the hypersurface differential equations.
+ * There are five, in sequence, deriving \f$\beta, Q, U, W,\f$ and \f$H\f$ For
+ * each of the five radial differential equations, first the products and
+ * derivatives on the right-hand side must be evaluated, then the full
+ * right-hand side of the equation must be computed, then the radial
+ * differential equation integrated. The equations have a hierarchical
+ * structure, so the result for \f$\beta\f$ feeds into the right-hand side for
```

Comment:
would right-hand side -> radial differential equation be correct?  If so, we find this to be more clear